src/components/dataset/pending-submissions.vue

```diff
@@ -91,17 +91,17 @@ const convert = ref(null);
     "explanation": {
       "userAction": "You are setting Entity creation to occur when Submissions are first received by Central.",
       "implication": {
-        "full": "Once this takes effect, Entities will no longer be generated when Submissions are marked Approved, including {records} we found that has neither been marked Approved nor Rejected. | Once this takes effect, Entities will no longer be generated when Submissions are marked Approved, including {records} we found that have neither been marked Approved nor Rejected.",
-        "records": "{count} record | {count} records"
+        "full": "You currently have {records} not marked Approved nor Rejected.",
+        "records": "{count} pending record | {count} pending records"
       }
     },
     "dontConvert": {
       "label": "I understand and this is not a problem for me.",
-      "description": "Change the setting and do nothing with the pending Submission. | Change the setting and do nothing with the pending Submissions."
+      "description": "Change the setting and do nothing with the pending Submission."
```

Comment:
Does this need to change? I think it wouldn't read quite right with "the pending Submission" (singular) when there's more than one pending submission.

If you wanted to make it static text and not pluralizable, I think always using "Submissions" in the plural would work a little better, something like:

```suggestion
      "description": "Change the setting and do nothing with pending Submissions."
```